webapp/app/actions.js

```diff
@@ -1,13 +1,17 @@
 import axios from 'axios'
 
-import * as User from '@core/user/user'
 import * as i18nFactory from '@core/i18n/i18nFactory'
 
-import { LoaderActions } from '@webapp/store/ui'
 import { SystemStatusState } from '@webapp/store/system'
 
 export const appPropsChange = 'app/props/change'
-export const appUserLogout = 'app/user/logout'
+
+const _fetchUserAndSurvey = async () => {
+  const {
+    data: { user, survey },
+  } = await axios.get('/auth/user')
+  return { user, survey }
+}
```

Comment:
we need to find a place for this but maybe the `initSystem` should dispatch `initUser` which fetch survey and user too

Reply:
ok in this case yes